### ISSUE TITLE
go:linkname must refer to declared function or variable

### ISSUE THREAD

**Issue by mozey** [mozey/watcher#7]
When trying to install the `watcher` cmd
```
go install github.com/mozey/watcher
```

it errors with lines like the following
```
go/src/github.com/mozey/watcher/vendor/golang.org/x/sys/unix/zsyscall_darwin_arm64.go... go:linkname must refer to declared function or variable
```

**Comment by mozey** [comment]
See this [comment](https://github.com/golang/go/issues/51706#issuecomment-1074480464), and [solution here](https://stackoverflow.com/a/72883302/639133): update the `golang.org/x/sys` package and vendor
```
go get -u golang.org/x/sys
go mod vendor
go mod verify
```

Fixed in dcc1a2c

**Comment by mozey** [comment]
See https://github.com/mozey/watcher/releases/tag/v0.1.1
